src/components/select/template.njk

```diff
@@ -0,0 +1,18 @@
+{% from "label/macro.njk" import govukLabel %}
+{% set labelArguments = {
+  text: params.label.text if params.label.text else params.label,
+  html: params.label.html,
+  hintText: params.hintText if params.hintText else params.hintHtml,
```

Comment:
This isn't quite right – it'd mean `hintHtml` would get passed through as `hintText` to the label component, which then wouldn't escape it.